spec/models/comment_spec.rb

```diff
@@ -82,4 +87,17 @@
       end
     end
   end
+
+  describe "#mentioned audiences" do
+    it "mentioning an audience should return corresponding audience" do
+      audience = Audience.all.sample
+      comment1 = create :comment, body: "Hey, @#{audience}"
+      comment2 = create :comment,
+                        body: "<p>Hey, <span class=\"mention\" data-mention-id=\"#{audience}\" label=\"#{audience}\">@#{audience}</span></p>",
```

Comment:
can you take the interpolation of audience out of this, and test it explicitly in each case please? IE:
```
body: "<p>Hey, <span class=\"mention\" data-mention-id=\"group\" label=\"group\">@group</span></p>"
```